janitor/functions.py

```diff
@@ -412,7 +412,7 @@ def remove_empty(df: pd.DataFrame) -> pd.DataFrame:
 @pf.register_dataframe_method
 @deprecated_alias(columns="column_names")
 def get_dupes(
-    df: pd.DataFrame, column_names: Union[str, Iterable[str], Hashable] = None
+    df: pd.DataFrame, column_names: Optional[Union[str, Iterable[str]]] = None
```

Comment:
I think Hashable was intentionally added, because that's _technically_ the type of any pandas column name. 

@hectormz I think you were involved in this; does removing the Hashable type simplify the codebase?

Reply:
I don't believe that I was involved with adding the `Hashable` typing. If I recall it was @dendrondal ? 

But I've been interested in this for some time since trying out `mypy` and it made a comment about a few complaints that were typed as `Hashable`.

Reply:
OK, my bad! Memory failing me. In any case, @VPerrollaz, could you add back `Hashable` please?

Reply:
@ericmjl @VPerrollaz I mean that perhaps the use of `Hashable` should still be considered. `mypy` seemed to say we were using it wrong in some instances...

Reply:
No problem putting back `Hashable`, I was a bit perplexed by the Hashable I must say, which is why I proposed adding a new type.